cor_threshold/code/clean_data.R

```diff
@@ -49,8 +55,9 @@ for (time in times) {
       "outcome",
       "Delta"
     )
-  keep <- data$Trt == 1 & data$Bserostatus == 0 & !is.na(data$wt)
 
+  keep <- data[[Earlyendpoint]] ==0 & data$Trt == 1 & data$Bserostatus == 0 & data$Perprotocol==1 & !is.na(data$wt) & data[[Event_Time_variable[[time]]]] >=7
```

Comment:
@youyifong, will event time variables be allowed to have `NA`'s for the early event individuals?  Or will their failure times be negative?

@Larsvanderlaan, if the former then could you confirm that this code executes as expected with `NA`'s?

Reply:
Nevermind, I see in #309 that we disallow `NA`'s in these variables.